### PR TITLE
refactor: cms.enabled is used to determine if inline forms are active

### PR DESCRIPTION
**Pull request by ncphillips**

### CODE REVIEW

File: packages/demo-gatsby/src/templates/blog-post.js [hunk 1]
```diff
@@ -42,14 +42,14 @@ import { useCMS } from "tinacms"
 const get = require("lodash.get")
 
 function InlineWysiwyg(props) {
-  const { status } = useInlineForm()
+  const cms = useCMS()
   const [{ InlineWysiwyg }, setEditor] = React.useState({})
 
   React.useEffect(() => {
     if (!InlineWysiwyg && status === "active") {
```
Comment (dwalkr) [inline packages/demo-gatsby/src/templates/blog-post.js:49]:
```suggestion
    if (!InlineWysiwyg && cms.enabled) {
```

Comment (ncphillips) [reply packages/demo-gatsby/src/templates/blog-post.js:49]: Weird, I thought I made that change

**Comment by kendallstrautman** [comment]
Should the individual fields call `useCMS` or could they still get passed a 'status' prop from `InlineField` based on`cms.enabled`?

**Comment by ncphillips** [comment]
They could but I'm not sure what the value of that indirection would be if the status is always going to be `active` when `cms.enabled`

**Comment by kendallstrautman** [comment]
Yeah I was thinking it could just be a boolean or something, so that only `InlineForm` calls `useCMS` and all the child fields don't have to. But I'm down with either way.

File: packages/react-tinacms-inline/src/inline-form.tsx [hunk 2]
```diff
@@ -40,29 +39,16 @@ export interface InlineFormState {
...
   const inlineFormState = React.useMemo(() => {
     return {
       form,
-      status,
       focussedField,
       setFocussedField,
-      activate: () => setStatus('active'),
-      deactivate: () => setStatus('inactive'),
     }
   }, [form, status, focussedField])
```
Comment (kendallstrautman) [inline packages/react-tinacms-inline/src/inline-form.tsx:53]:
```suggestion
  }, [form, focussedField])
```

**Pull request merged**